packages/web/src/components/AdminApplicationUpdateOAuthClient/index.jsx

```diff
@@ -62,24 +91,25 @@ function AdminApplicationUpdateOAuthClient(props) {
     return defaultValues;
   }, [auth?.fields]);
 
-  const defaultValues = useMemo(
-    () => ({
+  useEffect(() => {
+    setDefaultValues({
       name: adminOAuthClient?.data?.name || '',
       active: adminOAuthClient?.data?.active || false,
-      ...getAuthFieldsDefaultValues(),
-    }),
-    [adminOAuthClient, getAuthFieldsDefaultValues],
-  );
+      ...authFieldsDefaultValue,
+    });
+  }, [adminOAuthClient, authFieldsDefaultValue]);
 
   return (
-    <AdminApplicationOAuthClientDialog
+    <AdminApplicationOAuthClientUpdateDialog
       onClose={onClose}
       error={updateOAuthClientError}
       title={formatMessage('updateOAuthClient.title')}
       loading={isAdminOAuthClientLoading}
-      submitHandler={submitHandler}
+      submitAuthDefault={handleUpdateAuthDefaults}
```

Comment:
```suggestion
      submitAuthDefaults={handleUpdateAuthDefaults}
```

for the sake of consistency